libraries/Microsoft.Bot.Builder/TurnContext.cs

```diff
@@ -214,16 +208,14 @@ public bool Responded
         public Task<ResourceResponse[]> SendActivities(IActivity[] activities)
         {
             if (activities == null)
-            {
                 throw new ArgumentNullException(nameof(activities));
-            }
 
             if (activities.Length == 0)
             {
```

Comment:
Why does this if have {}, but the prior check (right above) is a single line if?